ddp-workspace/projects/toolkit/src/lib/components/dialogs/sessionWillExpire.component.ts

```diff
@@ -68,13 +68,11 @@ export class SessionWillExpireComponent implements OnInit, OnDestroy {
 
     public signOut(): void {
         this.renewNotifier.hideSessionExpirationNotifications();
-        this.session.disableTokenExpiredProcess();
         this.auth0.logout();
     }
 
     public renewSession(): void {
         this.renewNotifier.hideSessionExpirationNotifications();
-        this.session.disableTokenExpiredProcess();
         this.auth0.auth0RenewToken();
```

Comment:
Perhaps this caused an error... since we have additional time, we don't need to disableTokenExpiredProcess.
But honestly, I couldn't reproduce an error, I spent hours trying to reproduce locally and on osteo dev. Moreover, it really weird that only osteo affected by this issue, because angio, brain, mbc and osteo use the same component!
Do we have some special config for auth0/backend for the app which configures the possibility to renew the session? The apps use this method https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts#L203
cc @yufengwng, @zyme, @MocanaAtBroad